vllm/hpu/ops.py

```diff
@@ -11,87 +11,72 @@
 import habana_frameworks.torch as htorch
 from typing import List, Optional, Tuple
 
+import vllm.hpu.utils as hpu_utils
+
+
 def silu_and_mul(output, input):
-    htorch.core.mark_step()
     d = input.shape[-1] // 2
     silu = torch.nn.SiLU().to(input.device)
     x, y = torch.split(input, d, dim=-1)
     output.copy_(silu(x) * y)
-    htorch.core.mark_step()
```

Comment:
any reason for removing these?

Reply:
Basically, not needed. It's a very simple operation. There's no need to break the graph here.